app/models/user.rb

```diff
@@ -4,6 +4,11 @@ class User < ApplicationRecord
   has_many :applications, dependent: :destroy
   has_many :ecf_sync_request_logs, as: :syncable, dependent: :destroy
 
+  validates :full_name, presence: true
+  validates :email, presence: true, uniqueness: true, notify_email: true, on: :npq_separation # rubocop:disable Rails/UniqueValidationWithoutIndex
+  validates :uid, uniqueness: { allow_blank: true }
```

Comment:
maybe we can scope this to separation as well just incase? Im worried that this might still happen while the email validation is not set?

Reply:
Yeah can do; I checked the database and all the `uid` values were currently unique but that may not always be the case I guess 👍

Reply:
Actually I think we're OK to keep this one as there is a unique constraint in the DB:

https://github.com/DFE-Digital/npq-registration/blob/main/db/schema.rb#L362